### ISSUE TITLE
LangChain in AI Beta Hangs sometimes / cannot be stopped via UI

### ISSUE THREAD

**Issue by amenk** [n8n-io/n8n#7666]
**Describe the bug**

When using LangChain on the AI Beta the workflow hangs, does not finish and also cannot be stopped

**To Reproduce**
No always reproducible, but more often than not

1. Start workflow
2. Chain does not run through
3. Click stop workflow
4. All hangs, see the following GIF:

![n8n-llm all hangs](https://github.com/n8n-io/n8n/assets/1087128/e2711bc9-ac19-4632-934e-9d98754b5f98)

**Expected behavior**

All works, or at least an error appears if something is wrong with the ChatGPT API / and the workflow can be stopped

**Environment (please complete the following information):**

```
docker run -it --rm --name n8nai -p 5678:5678 -e N8N_LOG_LEVEL=debug -e N8N_LOG_OUTPUT=console -v ~/.n8nai:/home/node/.n8n docker.n8n.io/n8nio/n8n:ai-beta
```

- OS: 22.04
- n8n Version Image: `docker.n8n.io/n8nio/n8n                         ai-beta              f38ede559f01   5 days ago      754MB`
- Node.js Version docker
- Database system  docker
- Operation mode  ?

**Additional context**

On the console I see:

```
2023-11-09T09:39:00.614Z [Rudder] debug: in flush
2023-11-09T09:39:00.615Z [Rudder] debug: cancelling existing timer...
2023-11-09T09:39:00.615Z [Rudder] debug: cancelling existing flushTimer...
2023-11-09T09:39:00.615Z [Rudder] debug: batch size is 3
2023-11-09T09:39:20.865Z [Rudder] debug: in flush
2023-11-09T09:39:20.865Z [Rudder] debug: cancelling existing timer...
2023-11-09T09:39:20.865Z [Rudder] debug: queue is empty, nothing to flush
2023-11-09T09:39:37.470Z | debug    | Wait tracker querying database for waiting executions "{ file: 'WaitTracker.js', function: 'getWaitingExecutions' }"
2023-11-09T09:40:37.471Z | debug    | Wait tracker querying database for waiting executions "{ file: 'WaitTracker.js', function: 'getWaitingExecutions' }"
2023-11-09T09:40:51.250Z | debug    | Proxying request to axios "{ file: 'LoggerProxy.js', function: 'exports.debug' }"
2023-11-09T09:41:37.473Z | debug    | Wait tracker querying database for waiting executions "{ file: 'WaitTracker.js', function: 'getWaitingExecutions' }"
2023-11-09T09:42:37.475Z | debug    | Wait tracker querying database for waiting executions "{ file: 'WaitTracker.js', function: 'getWaitingExecutions' }"
```

After pressing Ctrl+C

```
2023-11-09T09:44:09.902Z [Rudder] debug: batch size is 1
Waiting for 1 active executions to finish...
 - Execution ID 393, workflow ID: enkm09FeKlhfSAlB
Waiting for 1 active executions to finish...
 - Execution ID 393, workflow ID: enkm09FeKlhfSAlB
```

**Comment by amenk** [comment]
PS: I got an error from the pipeline for this issue: https://github.com/n8n-io/n8n/actions/runs/6810063907

**Comment by amenk** [comment]
@OlegIvaniv Yes I also assume that there is some limit hit at OpenAI.

1. where can I see this?
2. Still the workflow should be stoppable without killing the container?

**Comment by OlegIvaniv** [comment]
@amenk You can check your usage in [OpenAI usage dashboard](https://platform.openai.com/usage)

**Comment by amenk** [comment]
@OlegIvaniv Yeah, sure. But I believe I don't see the current rate limits there. Was more thinking about the response headers. Does not even to be displayed with the "debug" log level.

Also it was my first run today, so I should not have hit limits. But it's all guessing ;-)

The main reason I opened the issue is because of the hard-hang, I believe that should never happen, even if an external API is limiting?

**Comment by amenk** [comment]
Is this considered a bug? Are there any workarounds?

**Comment by Joffcom** [comment]
We’re taking some time to review and tidy up our issue tracker. It looks like this issue hasn’t had any recent activity for the past 6 months. To keep our backlog focused on current priorities, we’re closing this issue for now.

That said, this doesn’t mean the issue is forgotten! If this is still relevant or you have more information to add, feel free to reopen it or create a new issue and we’d be happy to revisit it.

Thank you for understanding and for contributing to the project!
